libweb3jsonrpc/Test.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+	This file is part of cpp-ethereum.
+
+	cpp-ethereum is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	cpp-ethereum is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file Test.cpp
+ * @authors:
+ *   Dimitry Khokhlov <dimitry@ethdev.com>
+ * @date 2016
+ */
+
+#include "Test.h"
+#include <libethereum/ClientTest.h>
+
+using namespace std;
+using namespace dev;
+using namespace dev::rpc;
+
+Test::Test(eth::Client& _eth): m_eth(_eth) {}
+
+bool Test::test_mineBlocks(int _number, int _timestamp)
+{
+	asClientTest(&m_eth)->mineBlocks(_number, _timestamp);
+	return true;
+}
+
+bool Test::test_addBlock(std::string const& _rlp)
```

Comment:
I think it would be better to have `addBlocks(unsigned _number)` (i.e. adds `_number` empty blocks) and unrelated to that `setTimestamp(unsigned _timestamp)` which sets the timestamp for the pending block. I.e. if I first call `setTimestamp` and then use `eth_call` and request the timestamp in the code, it returns that timestamp.